### PR TITLE
Add ElasticityKernel

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by inducer** [comment]
Thanks, LGTM! Any idea why pytential is unhappy?

File: sumpy/kernel.py [hunk 1]
```diff
@@ -662,44 +664,62 @@ def get_pde_as_diff_op(self):
...
 
-class StokesletKernel(ExpressionKernel):
-    init_arg_names = ("dim", "icomp", "jcomp", "viscosity_mu_name")
+class ElasticityKernel(ExpressionKernel):
+    init_arg_names = ("dim", "icomp", "jcomp", "viscosity_mu_name",
+            "poisson_ratio")
 
-    def __init__(self, dim, icomp, jcomp, viscosity_mu_name="mu"):
+    def __new__(cls, dim, icomp, jcomp, viscosity_mu="mu", poisson_ratio="nu"):
+        if poisson_ratio == "0.5":
```
Comment (inducer) [inline sumpy/kernel.py:672]: Is a string really the best solution here? Could you explain?

Comment (isuruf) [reply sumpy/kernel.py:672]: No, it isn't. I tried pymbolic expressions, but it fails with,
```
TypeError: unsupported type for persistent hash keying: <class 'pymbolic.primitives.Variable'>
```
for `mu`.

**Comment by inducer** [comment]
Thanks!

File: sumpy/kernel.py (hunk 1 above)
Comment (inducer) [reply sumpy/kernel.py:672]: Thanks for making this happen.

**Pull request merged**